x/launch/client/cli/tx_request_add_vested_account.go

```diff
@@ -0,0 +1,56 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	codec "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func CmdRequestAddVestedAccount() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "request-add-vested-account [chainID] [coins] [options]",
+		Short: "Request to add a vested account",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			coins, err := sdk.ParseCoinsNormalized(args[1])
+			if err != nil {
+				return fmt.Errorf("failed to parse coins: %w", err)
+			}
+
+			delayedVesting, err := codec.NewAnyWithValue(&types.DelayedVesting{
```

Comment:
For `DelayedVesting` you have a value `Vesting` which are coins that will be vested at the time `EndTime`

Since for now we only support `DelayedVesting, can we have two additional positional argument:
- `vesting-coins`
- `vesting-end-time`